### DISCUSSION TITLE
Nothing happens when clicking change ID on qfield.cloud

### DISCUSSION THREAD

**Discussion by ThomasDV1984** [opengisch/QField#4991] [Bugs]
when collecting objects in the field, one object wasn't pushed. When I when to the changelog on qfield.cloud I could normally push on the change and see it on a map, but now nothing happens if i push on de link of the change ID. Is this someting new or a bug?

**Comment by chrysenteron** [comment]
I have the same problem. This is actually quite an important feature to me, since there's sometimes changes, that can't be applied or cause errors and I need to be able to see the details to make sure no data gets lost.

**Comment by suricactus** [comment]
Note this has been already reported https://github.com/opengisch/qfieldcloud/issues/849, fix is implemented and due to be released in the upcoming days.
